senza/manaus/utils.py

```diff
@@ -0,0 +1,15 @@
+from typing import Dict, Optional  # noqa: F401
+
+from botocore.exceptions import ClientError
+
+__all__ = ["extract_client_error_code"]
```

Comment:
It's also documentation about the public API. On pycharm, for example, variables and functions in __all__ appear first on autocompletion.